cmd/kola/spawn.go

```
@@ -87,7 +91,20 @@ func doSpawn(cmd *cobra.Command, args []string) error {
 
 	var someMach platform.Machine
 	for i := 0; i < spawnNodeCount; i++ {
-		mach, err := cluster.NewMachine(userdata)
+		var mach platform.Machine
+		if kolaPlatform == "qemu" && spawnMachineOptions != "" {
+			b, err := ioutil.ReadFile(spawnMachineOptions)
+			if err != nil {
+				return fmt.Errorf("Could not read machine options: %v", err)
+			}
+
+			var machineOpts qemu.MachineOptions
+			json.Unmarshal(b, &machineOpts)
```

Check for errors otherwise this will silently fail and not apply the options